uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -4366,6 +4367,61 @@ public function testTheTimeForFromAndToDateOfLeaveRequestIsAddedCorrectlyWhenLea
     $this->assertEquals($toDate->format('Y-m-d') ." 23:59:00", $leaveRequest->to_date);
   }
 
+  public function testTheTimeForFromAndToDateOfLeaveRequestStaysUntouchedWhenRequestIsTOIL() {
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'allow_accruals_request'   => true,
+      'max_leave_accrual'        => 10,
+      // The calculation unit is in days since otherwise a check for
+      // calculation unit in days will take action and this test will be invalid
+      'calculation_unit'         => 1,
+      'allow_accrue_in_the_past' => true
+    ]);
+    $expectedFromDate = '2016-01-08 01:23:45';
+    $expectedToDate = '2016-01-10 12:34:56';
+    $fromDate = new DateTime($expectedFromDate);
+    $toDate = new DateTime($expectedToDate);
```

Comment:
Why not just 
```
$fromDate = new DateTime('2016-01-08 01:23:45');
$toDate = new DateTime('2016-01-10 12:34:56');
``` 
You can always do `$fromDate->format('YmdHis')` anywhere you need the datetime.

Reply:
I would like to keep assertions as clean as possible. I would like to compare values to simple strings rather than datetime object formatted.

Reply:
Won't make it less clean if we keep them as the date objects